packages/base-driver/lib/basedriver/driver.ts

```diff
@@ -162,7 +163,12 @@ export class BaseDriver<
     // log timing information about this command
     const endTime = Date.now();
 
-    cmd = this.clarifyCommandName(cmd, args);
+    if (cmd === 'execute') {
+      const firstArg = args?.[0];
+      if (typeof firstArg === 'string' && firstArg.includes(':')) {
```

Comment:
we must not make any assumptions about whether an argument contains a colon.checking if it's a string (and maybe not empty) should suffice